data/so-Osma

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+anglicized_name: Somali
+native_name: Soomaali
```

Comment:
please change this to:
```
anglicized_name: Somali (Osmanya script)
native_name: Soomaali (Farta Cismaanya)
```
(I'm sorry that I didn't give enough information in my previous request about this.)

Reply:
Added in https://github.com/typekit/speakeasy/pull/43/commits/1f88ff197b42fd35727e3d77ba68b7adfa20b878.